Written improved README (#67)
hackclub/burrow: README.md

![License](https://img.shields.io/github/license/hackclub/burrow) ![Apple Build Status](https://img.shields.io/github/actions/workflow/status/hackclub/burrow/build-apple.yml?branch=main&label=macos%2C%20ios&logo=Apple) ![Crate Build Status](https://img.shields.io/github/actions/workflow/status/hackclub/burrow/build-rust.yml?branch=main&label=crate&logo=Rust)

> Burrow is an open source tool for burrowing through firewalls. It is built by teenagers at [Hack Club](https://hackclub.com/).

Reviewer: ```suggestion
Burrow is an open source tool for burrowing through firewalls, built by teenagers at [Hack Club](https://hackclub.com/).
```

I like this phrasing, it feels less stiff to me.

> It works through the app that allows for a quick and easy setup to open virtual interfaces and direct traffic through them. You can also use burrow directly from the command line. 

Reviewer: ```suggestion
`burrow` provides a simple command-line tool to open virtual interfaces and direct traffic through them. You can also use burrow directly from the command line. 
```

What're your thoughts on this?

Author: I like it, but the second part feels redundant given the change, might be a good idea to remove it

Author: Already implemented changes

> It works through the app that allows for a quick and easy setup to open virtual interfaces and direct traffic through them. You can also use burrow directly from the command line. 

Reviewer: I'm not sure on the wording here. Can we call it best-in-class when it's not even done yet?

Author: Yeah, guess you're right

## Contributing

Burrow is fully open source, you can fork the repo and start contributing easily. For more information and in-depth discussions, visit the `#burrow` channel on the [Hack Club Slack](https://hackclub.com/slack/), here you can ask for help and talk with other people interested in burrow! 

The project structure is divided in the following folders: 

> 
> [] To do, add folder directories
> 

Reviewer: ```suggestion

\```
Apple/ # Xcode project for burrow on macOS and iOS
burrow/ # Higher-level API library for tun and tun-async
tun/ # Low-level interface to OS networking
    src/
        unix/ # macOS and Linux code
        windows/ # Windows networking code
tun-async/ # Async interface to tun
\```

```

Had to escape the inner  backticks to make github keep it as a suggestion, but this will fill out this section

Author: Okok, I'll commit it

> 

Reviewer: Do you want to document the git lint rules here? Or should that go in a CONTRIBUTING.md?

Author: We can mention it briefly, and then provide details in the CONTRIBUTING.md

Reviewer: ```suggestion

Burrow is fully open source, you can fork the repo and start contributing easily. For more information and in-depth discussions, visit the `#burrow` channel on the [Hack Club Slack](https://hackclub.com/slack/), here you can ask for help and talk with other people interested in burrow! For more information on how to contribute, please see [CONTRIBUTING.md]

The project structure is divided in the following folders: 

[] To do, add folder directories

```

![License](https://img.shields.io/github/license/hackclub/burrow) ![Apple Build Status](https://img.shields.io/github/actions/workflow/status/hackclub/burrow/build-apple.yml?branch=main&label=macos%2C%20ios&logo=Apple) ![Crate Build Status](https://img.shields.io/github/actions/workflow/status/hackclub/burrow/build-rust.yml?branch=main&label=crate&logo=Rust)

Burrow is an open source tool for burrowing through firewalls, built by teenagers at [Hack Club](https://hackclub.com/).

> `burrow` provides a simple command-line tool to open virtual interfaces and direct traffic through them

Reviewer: ```suggestion
`burrow` provides a simple command-line tool to open virtual interfaces and direct traffic through them.
```

Last extremely minor nitpick, just for consistency
